debsources/app/copyright/templates/copyright/index.html

```diff
@@ -15,31 +16,43 @@
 
 <div id="indexmenu">
 
-<h1 class="h1mainpage">{{ self.title() }}</h1>
-<h3><em>All Debian license are belong to us</em> &emsp;
-  <small>&mdash; Anonymous</small>
-  <sup><a href="https://en.wikipedia.org/wiki/All_your_base_are_belong_to_us"><span style="font-family: monospace; color: black;">[^]</span></a></sup>
-</h3>
-<p>Browse through the licenses of the
-  <a href="https://www.debian.org">Debian</a> operating system.
-  <a href="{{ url_for('doc.doc_overview') }}">Read more…</a></p>
-<table>
-  <tr>
-    <td id="browse">
-      <p><strong>Browse</strong> by prefix</p>
-      <p>{{ macros.render_packages_prefixes(packages_prefixes, request) }}</p>
-    </td>
-    <td id="search">
+  <div class="row">
+    <h1 class="h1mainpage">{{ self.title() }}</h1>
+    <h3 class="hidden-xs">
+      <em>All Debian license are belong to us</em> &emsp;
+      <small>&mdash; Anonymous</small>
+      <sup>
+        <a href="https://en.wikipedia.org/wiki/All_your_base_are_belong_to_us">
+          <span style="font-family: monospace; color: black;">[^]</span>
+        </a>
+      </sup>
+    </h3>
+
+    <div class="col-xs-2"></div>
+    <div class="col-xs-8">
+      <p>Browse through the licenses of the
+      <a href="https://www.debian.org">Debian</a> operating system.
+      <a href="{{ url_for('doc.doc_overview') }}">Read more…</a></p>
+    </div>
+    <div class="col-xs-2"></div>
```

Comment:
if what you are trying to achieve is centering the previous thing then you should have a look at offset/pull/push classes of columns that enable this with less and clearer code. Unless i am missing something, please mention so :)